src/AutoRest.CSharp/Common/Output/Models/Types/BuildContext.cs

```diff
@@ -10,23 +10,20 @@ namespace AutoRest.CSharp.Output.Models.Types
 {
     internal class BuildContext
     {
-        public BuildContext(CodeModel codeModel, SourceInputModel? sourceInputModel) : this(codeModel, sourceInputModel, Configuration.LibraryName, Configuration.Namespace)
+        public BuildContext(InputNamespace inputNamespace, SourceInputModel? sourceInputModel) : this(inputNamespace, sourceInputModel,Configuration.LibraryName, Configuration.Namespace)
```

Comment:
would this PR be an oppotunity to remove this class? Will removing this class cause a lot of effort?

Reply:
Given the existing changed files, I will not remove it in this PR. Let's see if it gets easier after this PR